package.json

```diff
@@ -30,47 +30,53 @@
     "**/*.{css,less}": "stylelint"
   },
   "devDependencies": {
-    "@babel/eslint-parser": "^7.21.8",
-    "@babel/preset-react": "^7.22.3",
-    "@changesets/cli": "^2.26.1",
+    "@babel/eslint-parser": "^7.25.9",
+    "@babel/preset-react": "^7.25.9",
+    "@changesets/cli": "^2.27.9",
+    "@eslint/compat": "^1.2.1",
+    "@eslint/eslintrc": "^3.1.0",
+    "@eslint/js": "^9.13.0",
     "@ima/plugin-cli": "^19.1.2",
     "@ima/testing-library": "^19.10.0",
-    "@swc/jest": "^0.2.26",
+    "@swc/jest": "^0.2.36",
     "@testing-library/dom": "^10.4.0",
-    "@testing-library/jest-dom": "^6.4.8",
-    "@testing-library/react": "^16.0.0",
-    "@types/jest": "^29.5.1",
+    "@testing-library/jest-dom": "^6.6.2",
+    "@testing-library/react": "^16.0.1",
+    "@types/eslint__js": "^8.42.3",
+    "@types/jest": "^29.5.14",
     "@types/node": "^20.2.5",
-    "@typescript-eslint/eslint-plugin": "^5.59.8",
-    "@typescript-eslint/parser": "^5.59.8",
-    "eslint": "^8.41.0",
-    "eslint-config-prettier": "^8.8.0",
-    "eslint-plugin-import": "^2.27.5",
-    "eslint-plugin-jest": "^27.2.1",
-    "eslint-plugin-jsdoc": "^46.0.0",
-    "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-react": "^7.32.2",
-    "husky": "^8.0.3",
-    "jest": "^29.5.0",
-    "jest-environment-jsdom": "^29.5.0",
+    "@typescript-eslint/eslint-plugin": "^8.11.0",
+    "@typescript-eslint/parser": "^8.11.0",
```

Comment:
Do we still need to define `@typescript-eslint/eslint-plugin` and `@typescript-eslint/parser`? It wasn't mentioned in the `typescript-eslint` install instructions. It is also a direct dependency of `typescript-eslint` and all these packages seem to have matching version

Reply:
`@typescript-eslint/parser` and `@typescript-eslint/eslint-plugin` removed from devDependencies